modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -114,6 +114,10 @@ def handle264555
         lgy_response = LGY::Service.new.post_grant_application(payload: form.as_payload)
         reference_number = lgy_response.body['reference_number']
         status = lgy_response.body['status']
+        Rails.logger.info(
+          'Simple forms api - sent to lgy',
+          { form_number: params[:form_number], status:, reference_number: }
+        )
```

Comment:
Would we like to log the response body as well here? Also perhaps the specific context in why something was sent to loan guarantee?

Reply:
Hmm I think we'd need to know all the keys we want from the response body or else we'd be logging a hash. And if we log a hash then it will likely be less readable in the logs. I think grabbing the `status` and `reference_number` is sufficient. I'm not even sure if there are other keys in the response body that would be helpful.

In terms of why something was sent to LGY, it will be submissions of form 26-4555. I don't know of any other forms that interact with that API but if they crop up in the future, we can deal with it then, I think.